crates/uv/src/lib.rs

```diff
@@ -1,17 +1,17 @@
+use anstream::eprintln;
+use anyhow::{bail, Result};
+use clap::error::{ContextKind, ContextValue};
+use clap::{CommandFactory, Parser};
+use owo_colors::OwoColorize;
+use settings::PipTreeSettings;
 use std::borrow::Cow;
 use std::env;
 use std::ffi::OsString;
 use std::fmt::Write;
 use std::io::stdout;
 use std::path::Path;
 use std::process::ExitCode;
-
-use anstream::eprintln;
-use anyhow::{bail, Result};
-use clap::error::{ContextKind, ContextValue};
-use clap::{CommandFactory, Parser};
-use owo_colors::OwoColorize;
-use settings::PipTreeSettings;
+use std::sync::atomic::Ordering;
```

Comment:
I normally don't comment on this stuff but can we keep this as `std` then third-party? (Only because this moves a lot of imports around unnecessarily.)